CSharp/Library/Microsoft.Bot.Builder/ConnectorEx/IBotData.cs

```diff
@@ -67,5 +68,12 @@ public interface IBotData
         /// </summary>
         /// <param name="cancellationToken">The cancellation token.</param>
         Task FlushAsync(CancellationToken cancellationToken);
+
+        /// <summary>
+        /// returns BotStateData for the bot. Repeat calls until the continuation token is empty. />
+        /// </summary>
+        /// <param name="continuationToken">The continuation token.</param>
+        Task<BotStateDataResult> ExportStateDataAsync(string continuationToken);
```

Comment:
>ExportStateDataAsync [](start = 33, length = 20)

does it makes sense to add this interface outside of the connector's REST layer?

Reply:
Agreed. I'm removing this.

---
In reply to: [177233734](https://github.com/Microsoft/BotBuilder/pull/4373#discussion_r177233734) [](ancestors = 177233734)